server/api.js

```diff
@@ -273,6 +275,10 @@ router.post("/tender", async (req, res) => {
 		newErrors.push("Please select at least one skill.");
 	}
 
+	if (!buyerId) {
+		newErrors.push("Buyer ID is required.");
+	}
+
```

Comment:
This is unnecessary, buyerId will always be present, otherwise they can't access the endpoint